python/tox.ini

```diff
@@ -52,10 +52,13 @@ deps =
     black
     isort
     autoflake
+    pylint
+    pyenchant
 commands =
     autoflake -r --check --ignore-init-module-imports --remove-all-unused-imports src tests
     isort --profile black --check-only src tests
     black --line-length 130 --check --diff src tests
+    pylint --disable all --enable spelling --spelling-dict en_US --spelling-private-dict-file spellcheck-dictionary.txt en_US src tests
```

Comment:
Question / Nit: Do we need to specify `en_US` twice? Specifically for the 2nd argument after `--spelling-private-dict-file`.

Reply:
Great catch @kbendick! That's just me being messy with copy and paste!